workers/data_refinery_workers/processors/tximport.R

```diff
@@ -59,5 +59,3 @@ lstpm_df <- as.data.frame(txi_length_scaled$counts)
 lstpm_df <- tibble::rownames_to_column(lstpm_df, var = "Gene")
 tpm_filename <- file.path(exp_dir, "gene_lengthScaledTPM.tsv")
 readr::write_tsv(lstpm_df, path = tpm_filename)
-# Compress this output file
-R.utils::gzip(tpm_filename)
```

Comment:
Since this line is deleted, `R.uitls` package is not needed, so we can also delete the following line:
https://github.com/AlexsLemonade/refinebio/blob/220537e621d6355b2e54faa125ee84edeef3b55e/workers/install_tximport.R#L12